### DISCUSSION TITLE
Использование nfqws только для отдельного сегмента

### DISCUSSION THREAD

**Discussion by ab5784** [Anonym-tsk/nfqws-keenetic#4] [Делимся опытом]
Можно ли  сделать так, чтобы nfqws использовалась не для всех LAN-клиентов, а только для одного определённого сегмента домашней сети?

**Comment by Anonym-tsk** [comment]
Надо ковырять как кинетик маркирует трафик по сегментам. Теоретически, это возможно.

**Reply by ab5784** [reply]
Я начинал эксперименты с tpws, там это можно было задать в конфиге `/opt/etc/init.d/S51tpws`:
`iptables -t nat -A PREROUTING -i br0 -p tcp --dport 443 -j REDIRECT --to-port 999`
заменяя **`br0`** на интерфейс, "привязанный" к сегменту.
С tpws сначала работало ОК, но потом опять стало тупить ужасно, долго картинки превьюшек подгружает и т. д.
Если можно в nfqws что-то подобное подредактировать - то где?

**Reply by Anonym-tsk** [reply]
Там же в iptables. Там сейчас -o $IFACE указывается. Возможно, надо еще -i с вашим интерфейсом добавить. Не проверял.

**Reply by ab5784** [reply]
ОК, спасибо, попробую поиграться.

**Reply by red-ut-dm** [reply]
> ОК, спасибо, попробую поиграться.

Поделитесь информацией пожалуйста, получилось настроить?

**Reply by LimpingK** [reply]
> Поделитесь информацией пожалуйста, получилось настроить?

По-идее это проще сделать через политику доступа, как в мануале:

> На Keenetic/Netcraze можно создать политику доступа NFQWS (Приоритеты подключений – Политики доступа в интернет) и после перезапуска nfqws-keenetic будет работать только для устройств из этой политики.
Не забудьте поставить галочку на интерфейсе провайдера в созданной политике.

```
# Политика доступа (только для Keenetic/Netcraze)
POLICY_NAME="nfqws"

# Режим работы политики доступа
# 0 - обрабатывается трафик только для устройств в политике
# 1 - обрабатывается трафик для всех устройств, кроме добавленных в политику
POLICY_EXCLUDE=0|1
```
